spec/jobs/rapid_ready_for_decision/form526_asthma_job_spec.rb

```diff
@@ -10,20 +10,30 @@
     Sidekiq::Worker.clear_all
   end
 
-  let(:submission) { create(:form526_submission, :with_uploads) }
+  let!(:user) { FactoryBot.create(:disabilities_compensation_user, icn: '2000163') }
+  let(:auth_headers) do
+    EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
+  end
+  let(:saved_claim) { FactoryBot.create(:va526ez) }
+  let(:submission) do
+    create(:form526_submission, :with_uploads,
+           user_uuid: user.uuid,
+           auth_headers_json: auth_headers.to_json,
+           saved_claim_id: saved_claim.id,
+           submitted_claim_id: '600130094')
+  end
```

Comment:
All this extra setup is to mimic the setup in Form526HypertensionJobSpec which allows the VCR cassette to just work.

I do think we're probably due for a VCR cassette refactor sometime in the near future.

Reply:
Hmmm, interesting. Are you saying the FactoryBot for `form526_submission` isn't sufficient for enabling VCR? Why not?

Reply:
Okay, it looks like I can get rid of the saved claim, but gotta keep the user (for their ICN). Pushing an update now.